### ISSUE TITLE
Add Hotkey for Favorites

### ISSUE THREAD

**Issue by Lamaman71** [MrZammler/iGame#238]
Hi, 

I wanted to ask if you might consider adding a hotkey to add a game to the favorites in the next release. Adding them right now involves a right click into the top part of the interface, then a menu selection and a dialog box. Thanks!

**Comment by walkero-gr** [comment]
@Lamaman71 
Thank you for your proposal. I have thought of that, but the problem is that even if this was available, how would the user get feedback on his action? I mean, the user selects an item from the list, they click the keyboard hotkey. Then what? How do they know that this was applied?
Currently, the process might not be ideal, but it provides that information.

**Comment by Lamaman71** [comment]
Hi - thanks for the quick reply.

I appreciate your concerns too - but perhaps you could colour code or
Bold/Italic the Favourites in the list?

I love iGame but since the new release I cant even right click on the game
name to modify it - you now need to left click to select the title, move
the cursor to an area at the right of the list, then right click, select
Game -> Properties...its alot I feel for something that EVERYONE must be
doing to make the list more manageable. Even adding an option to simply
Right Click a game title and selecting it as a Favourite from that menu
would be an improvement.

For now, I send the whole CSV to my PC and use Notepad++ to edit the titles
in there and send it back to the Amiga. Not ideal but MUCH faster overall.

Anyway - awesome work, love you keep it up to date too - thanks so much.

Mark

**Comment by walkero-gr** [comment]
What you need to do is select the title > Right click at the top screen menu bar and then select the game properties. I understand that this might be time-consuming, and I am looking for ways to make it faster. But it was always like that. This didn't change with the latest versions. The other thing you could do is to use the RAmiga+P shortcut to open the properties after you click on a title.

About adding extra items on the context menu (right click over the title) is something I am thinking to do. I have already looked into it, and it might be in one of the next versions.

**Comment by Lamaman71** [comment]
Thats great - and thanks for taking time to correspond with me.

Just an FYI but the right Amiga + P option only works when you select the
title, then left click to select the right hand pane, then Amiga + P.

**Comment by walkero-gr** [comment]
> Just an FYI but the right Amiga + P option only works when you select the title, then left click to select the right hand pane, then Amiga + P.

Yeah, that is a bug of MUI3.8 and NList. It appears that it needs to lose the focus to access the shortcuts. Using MUI 5 this is working as it should. I do not remember if you first press TAB if that helps.

I will certainly have a look at how this can get better.
